GPU/GLES/TextureCacheGLES.cpp

```diff
@@ -925,7 +930,7 @@ void TextureCacheGLES::LoadTextureLevel(TexCacheEntry &entry, ReplacedTexture &r
 			// TODO: We really, really should avoid calling glGetError.
 			GLenum err = glGetError();
 			if (err == GL_OUT_OF_MEMORY) {
-				WARN_LOG_REPORT(G3D, "Texture cache ran out of GPU memory; switching to low memory mode");
+				WARN_LOG(G3D, "Texture cache ran out of GPU memory; switching to low memory mode");
```

Comment:
I kinda liked reporting this in case we suddenly started causing this more often, or if a setting was causing a leak.

-[Unknown]

Reply:
Yeah, fair enough.